File: SphrLibrary/Entities/OpenmHealth/BloodPressure.cs
using System.Runtime.Serialization;

namespace SphrLibrary.Entities.OpenmHealth
{
    /// <summary>
    /// Open mHealth 血圧データのリソースを表します。
    /// </summary>
    [DataContract()]
    [Serializable()]
    public class BloodPressure : OmhEntityBase<BloodPressureBody>
    {
        public BloodPressure() : base() { }
    }
}

File: References/Java/SphrLibrary-Java/lib/src/main/java/sphrlibrary/entities/openmhealth/BloodPressure.java
package sphrlibrary.entities.openmhealth;

import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.ToString;

/** 血圧データ */
@Data
@NoArgsConstructor
@ToString(callSuper = true)
@EqualsAndHashCode(callSuper = true)
public class BloodPressure extends OmhEntityBase<BloodPressureBody> {

}
